src/components/Market section/data.js

```diff
@@ -0,0 +1,20 @@
+import token from "/Market section/token.svg";

```

Comment:
use `camelCase` or something but dont put spaces in folder or file name
`Market Section` could be written as `MarketSection` or `marketSection`